### PR TITLE
Updated the cccp.yml file template to include more information.

### PR DESCRIPTION
**Pull request by mohammedzee1000**
The template cccp.yml file should contain more information about the container pipeline
itself, and should ideally be used in all the target repositores.

### CODE REVIEW

File: cccp.yml [hunk 1]
```diff
@@ -1,17 +1,24 @@
-#place this file in your nulecule dir and call it cccp.yml
+# This is for the purpose of building containers on the CentOS Community Container 
+# Pipeline. The containers are built, tested and delivered to registry.centos.org and
+# lifecycled as well. A corresponding entry must exist in the container index itself, 
+# located at https://github.com/CentOS/container-index/tree/master/index.d
+#  You can know more at the following links:
+# * https://github.com/CentOS/container-pipeline-service/blob/master/README.md
+# * https://github.com/CentOS/container-pipeline-service/blob/master/README.md
```
Comment (navidshaikh) [inline cccp.yml]: Line no 6 and line no 7 are repetitions.

File: cccp.yml [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
+#test-script     :
+#build-script    :
+#delivery-script :
+#docker-index    : True
+#custom-delivery : False
+#local-delivery  : True
+#Upstreams       :
+#        - ref           :
+#          url           :
+#desired-tag     : null
```
Comment (navidshaikh) [inline cccp.yml]: Why do we need to comment the job info ?

File: cccp.yml (hunk 1 above)
Comment (mohammedzee1000) [inline cccp.yml]: corrected

**Comment by mohammedzee1000** [comment]
Also, another reason why should those lines be commented is that in the index validation ci, i have it setup in such a way that except for job-id, if any of the other values are present, they are validated as well. For example, if someone specifies a build-script, then i actually check if the repository contains the specified build script and so on.

@navidshaikh does that answer your question about commenting those lines?

**Comment by navidshaikh** [comment]
> A reason why should those lines be commented is that in the index validation ci, i have it setup in such a way that except for job-id, if any of the other values are present, they are validated as well. For example, if someone specifies a build-script, then i actually check if the repository contains the specified build script and so on.

IMO, the tradeoff should be that CI scripts tweaked in a way to ignore *this** specific file in repository to not run validation for, instead of commenting out the lines for a template.

**Comment by mohammedzee1000** [comment]
Updated, also removed desired-tag as its moved to the index itelf

**Comment by mohammedzee1000** [comment]
@navidshaikh updated

**Pull request merged**